utils.js

```diff
@@ -0,0 +1,22 @@
+export const isElementAlreadyRegistered = (elName) => {
```

Comment:
Add JSDoc to this method.

Identifies whether a WebComponent with a name is already registered? Check is performed in both: `customElements` and `Polymer.telemetry` registries.